packages/chart/docs/QuartileCandlestick.md

```diff
@@ -0,0 +1,39 @@
+# QuartileCandlestick
+
+```meta
+{
+    "source": "https://github.com/hpcc-systems/Visualization/blob/master/packages/chart/src/QuartileCandlestick.ts#L4",
+    "extends": "SVGWidget"
+}
+```
+
+
+```sample-code
+import { QuartileCandlestick } from "@hpcc-js/chart";
+
+new QuartileCandlestick()
+    .target("target")
+    .columns(["Min","25%","50%","75%","Max"])
+    .data([122,315,456,987,1354])
+    .lineWidth(10)
+    .candleWidth(80)
+    .textPadding(8)
+    .labelFontSize(14)
+    .valueFontSize(14)
+    .lineColor("#333")
+    .innerRectColor("white")
+    .upperTextRotation(-60)
+    .lowerTextRotation(-60)
+    .edgePadding(0)
+    .roundedCorners(0)
+    .render()
+    ;
+
+```
+
+
```

Comment:
Needs more verbiage - also first sample should generally be one with no overrides of PPs?